### PR TITLE
documentation: ZENKO-1046_add_Ceph_backend

### PR DESCRIPTION
**Pull request by wabernat**
This PR introduces Ceph / Ceph-RADOS as appropriate (i.e., wherever other backends are mentioned) throughout the documentation.

This fixes ZENKO-1046

### CODE REVIEW

**Comment by wabernat** [comment]
@bert-e approve

File: docs/docsource/Operation-Architecture/Introduction/Conceptual_Framework.rst [hunk 1]
```diff
@@ -6,12 +6,12 @@ control, manage, replicate, and visualize data in a multi-cloud context.
...
 Fundamental building blocks of Zenko’s replication framework are the
 bucket, the location, and the cloud service. End users (people or
-services) store data as files in buckets. Administrators are free to
+services) store data as files in buckets. Administrators are free to
 configure and name buckets as they please—to the admin, buckets are
 essentially tenants aggregated in a cloud storage deployment. From an
 end user’s perspective, a bucket simply appears as a networked storage
-location, for example, “accounting,” “bioinformatics-lab,”
-“daily-video-01-07-2019,” or any other organizational unit that makes
+location, for example, “accounting,” “bioinformatics-lab,”
```
Comment (philipyoo) [inline docs/docsource/Operation-Architecture/Introduction/Conceptual_Framework.rst:13]: minor: quotes after the commas

**Review by philipyoo — APPROVED** [review]

**Comment by wabernat** [comment]
@bert-e reset

File: docs/docsource/Operation-Architecture/Introduction/supported_clouds.rst [hunk 2]
```diff
@@ -15,10 +15,11 @@ the site level to the following supported private and public clouds:
 -  Amazon-S3-based private clouds:
 
    - Scality RING S3 Connector
+   - Red Hat Ceph
```
Comment (tmacro) [inline docs/docsource/Operation-Architecture/Introduction/supported_clouds.rst]: Maybe change this to `Ceph RADOS Gateway`? Don't want user to get the idea that we also support Ceph's block storage

**Review by tmacro — APPROVED** [review]

File: docs/docsource/Operation-Architecture/Introduction/Conceptual_Framework.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/Operation-Architecture/Introduction/Conceptual_Framework.rst:13]: Comma inside the quote is a typographical convention, but this is a bit of a corner case/judgment call. If it were a list of parameters that have to be entered verbatim, I would change it for sure. However, this is describing a user's experience, so I'm defaulting to a Chicago running list style.

File: docs/docsource/Operation-Architecture/Introduction/supported_clouds.rst (hunk 2 above)
Comment (wabernat) [reply docs/docsource/Operation-Architecture/Introduction/supported_clouds.rst]: Good spot.

**Review by ssalaues — APPROVED** [review]

**Comment by ssalaues** [comment]
@wabernatScality It looks like commit lint doesn't like your last commit message otherwise it's good to go

**Comment by wabernat** [comment]
That oughta fix it...

**Comment by ssalaues** [comment]
@bert-e bypass_build_status

**Pull request merged**